src/Service.Tests/SqlTests/RestApiTests/Insert/MsSqlInsertApiTests.cs

```diff
@@ -261,6 +273,67 @@ public async Task TestCleanup()
 
         #endregion
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a timestamp field is missing from the request body.
```

Comment:
nit: i don't think the focus of the comment and testname should be on "timestamp". The current naming implies that there is something special about the timestamp field, when the test should be validating the behavior of readonly fields. 

Comment should describe why the missing field is significant and the effect you expect the missing field to have on DAB when Dab evaluates the request.

Reply:
The field 'row_version' doesn't look to be auto-generated/calculated. So including this field would be fine too.

Reply:
row_version is a timetstamp field which is read-only is MsSql. I have added details in the PR desc and the comment here in the linked issue for MsSql: https://github.com/Azure/data-api-builder/issues/1453#issuecomment-1637775760

Reply:
<img width="650" alt="image" src="https://github.com/Azure/data-api-builder/assets/34566234/e27de56a-0639-4dbe-a4dc-d32b6aeeed33">
We can't insert explicit values into timetstamp columns.

<img width="160" alt="image" src="https://github.com/Azure/data-api-builder/assets/34566234/8ba0c929-d361-45e5-a576-18692155dda3">
The example you linked seems to work but the value: `null` does not take effect even in that case.

Reply:
> The timestamp syntax is deprecated. This feature will be removed in a future version of Microsoft SQL Server. Avoid using this feature in new development work, and plan to modify applications that currently use this feature.
[MsLearn](https://learn.microsoft.com/en-us/sql/t-sql/data-types/rowversion-transact-sql?view=sql-server-ver16#:~:text=The%20timestamp%20syntax%20is%20deprecated.%20This%20feature%20will%20be%20removed%20in%20a%20future%20version%20of%20Microsoft%20SQL%20Server.%20Avoid%20using%20this%20feature%20in%20new%20development%20work%2C%20and%20plan%20to%20modify%20applications%20that%20currently%20use%20this%20feature.)

Should not use `rowversion` for this pr to avoid introducing deprecated features.

Reply:
The screenshot is from main branch. On my branch, it will be 400 yes.